test/src/test/groovy/hudson/model/AbstractProjectTest.groovy

```diff
@@ -594,4 +596,37 @@ public class AbstractProjectTest extends HudsonTestCase {
         }
         return con
     }
+
+    @Issue("JENKINS-27549")
+    public void testLoadingWithNPEOnTriggerStart() {
+        AbstractProject project = jenkins.createProjectFromXML("foo", getClass().getResourceAsStream("AbstractProjectTest/npeTrigger.xml"))
+
+        assert project.triggers().size() == 1
+    }
+
+    @TestExtension
```

Comment:
Can pass the test name `"testLoadingWithNPEOnTriggerStart"` as a parameter to ensure this extension is loaded just for the one test.